packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -816,9 +882,13 @@ class ListView extends BoxScrollView {
              ? itemBuilder(context, itemIndex)
              : separatorBuilder(context, itemIndex);
          },
-         childCount: math.max(0, itemCount * 2 - 1),
+         childCount: math.max(0, itemCount * 2 - 1), // same calculation as in semanticChildCount below.
```

Comment:
There is no way to call a static method or something like that here and below to have the calculation only in one place?

Reply:
Done